bin/fuel-core/chainspec/local-testnet/chain_config.json

```diff
@@ -45,255 +45,255 @@
-              "base": 2,
-              "units_per_gas": 1282
+              "base": 3,
+              "units_per_gas": 542
             }
           },
           "mcli": {
             "LightOperation": {
-              "base": 2,
-              "units_per_gas": 1250
+              "base": 3,
+              "units_per_gas": 537
             }
           },
           "mcp": {
             "LightOperation": {
-              "base": 3,
-              "units_per_gas": 385
+              "base": 4,
+              "units_per_gas": 435
             }
           },
           "mcpi": {
             "LightOperation": {
-              "base": 7,
-              "units_per_gas": 585
+              "base": 8,
+              "units_per_gas": 511
             }
           },
           "meq": {
             "LightOperation": {
-              "base": 2,
-              "units_per_gas": 1234
+              "base": 4,
+              "units_per_gas": 526
             }
           },
           "retd_contract": {
             "LightOperation": {
-              "base": 227,
-              "units_per_gas": 5
+              "base": 581,
+              "units_per_gas": 2
             }
           },
           "s256": {
             "LightOperation": {
-              "base": 25,
-              "units_per_gas": 5
+              "base": 59,
+              "units_per_gas": 2
             }
           },
           "scwq": {
             "HeavyOperation": {
-              "base": 5666,
-              "gas_per_unit": 6628
+              "base": 13638,
+              "gas_per_unit": 16781
             }
           },
           "smo": {
             "LightOperation": {
-              "base": 14635,
-              "units_per_gas": 3
+              "base": 38428,
+              "units_per_gas": 1
             }
           },
           "srwq": {
             "HeavyOperation": {
-              "base": 245,
-              "gas_per_unit": 243
+              "base": 548,
+              "gas_per_unit": 549
             }
           },
           "swwq": {
             "HeavyOperation": {
-              "base": 5661,
-              "gas_per_unit": 5776
+              "base": 13692,
+              "gas_per_unit": 14509
             }
           },
           "epar": {
             "HeavyOperation": {
-              "base": 69000,
-              "gas_per_unit": 52000
+              "base": 214810,
+              "gas_per_unit": 157836
             }
           },
           "contract_root": {
             "LightOperation": {
-              "base": 24,
-              "units_per_gas": 3
+              "base": 60,
+              "units_per_gas": 1
             }
           },
           "state_root": {
             "HeavyOperation": {
-              "base": 189,
-              "gas_per_unit": 96
+              "base": 475,
+              "gas_per_unit": 241
             }
           },
-          "new_storage_per_byte": 63,
+          "new_storage_per_byte": 1,
```

Comment:
It would be nice to use hardcoded value here. @rymnc Is that possible somehow to update it?=D Maybe we could make it part of the CI, or make it part of the `collect` binary

Reply:
hmm, we can try. idk how to do it but i can check